<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestisciIngressi</class>
 <widget class="QWidget" name="GestisciIngressi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1071</width>
    <height>706</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1071</width>
    <height>706</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1071</width>
    <height>706</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestisci Ingressi</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1071</width>
     <height>706</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>GestisciIngresso.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="ingressiConAbbonamento">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>250</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
background-color: lightgrey;
border-radius: 15px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="ingressiSenzaAbbonamento">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>340</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
background-color: lightgrey;
border-radius: 15px;
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_inserisciIngresso">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>560</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>INSERISCI 
 INGRESSO</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="GestisciIngresso.qrc"/>
 </resources>
 <connections/>
</ui>
